pydis_site/apps/content/resources/guides/pydis-guides/contributing/cloning-repository.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Cloning a Repository
+description: A guide to cloning git repositories.
+icon: fab fa-github
+---
+
+> **Note:** The process varies depending on your choice of code editor / IDE, so refer to one of the following guides:
+
+- [Cloning with PyCharm](#cloning-with-pycharm)
+- [Cloning with the command line](#cloning-with-the-command-line)
```

Comment:
Would this be better as a table of contents or would that make the note confusing?